src/Microsoft.ML.CpuMath/AlignedArray.cs

```diff
@@ -110,15 +108,21 @@ private void MoveData(int newBase)
             }
         }
 
-        public void CopyTo(Span<Float> dst, int index, int count)
+        public void CopyTo(Span<float> dst)
+        {
+            Contracts.Assert(dst != null);
+            Items.AsSpan().CopyTo(dst);
+        }
+
+        public void CopyTo(Span<float> dst, int index, int count)
         {
             Contracts.Assert(0 <= count && count <= _size);
             Contracts.Assert(dst != null);
```

Comment:
double-checking that this is intentional?

Reply:
Must be. I did not touch this code, except Float -> float!

---
In reply to: [243714825](https://github.com/dotnet/machinelearning/pull/1901#discussion_r243714825) [](ancestors = 243714825)